CHANGELOG.rst

```diff
@@ -8,9 +8,18 @@ CHANGELOG
 
 2.1.0
 =====
-* RAID support
-* EFS support
-* AWS Batch Multinode Parallel Support
+* Add support for RAID 0 and 1.
+* Add Elastic File System (EFS) support.
+* Add AWS Batch Multinode Parallel support.
+* Add support for Stockholm region (`eu-north-1`).
+* Add `--env` and `--env-blacklist` options to the `awsbsub` command to export environment variables
+  in the job environment.
+* Add `--input-file` option to the `awsbsub` command to stage-in input files from the client.
+* Export `PCLUSTER_JOB_S3_URL` variable in the job environment to be used for stage-in/out files
```

Comment:
Add new `PCLUSTER_JOB_S3_URL` variable to the job execution environment pointing to the S3 URL used for job data stage-in/-out